fullstack-curriculum/mod-1-javascript-fundamentals/3-control-flow.md

```diff
@@ -7,24 +7,167 @@ Follow along with code examples [here](https://github.com/The-Marcy-Lab-School/1
 ## Slides
 
 {% embed url="https://docs.google.com/presentation/d/102SIHvxRHHJ-_979yJdyq-3dI5NEVSME0uyBrSmgvrk/embed?start=false&loop=false&delayms=3000" %}
+
+## Overview
+
+**Objectives**
+
+**Key Terms**
+
+## Flow Control Review
+- Programs usually change their behavior based on inputs.
+- Conditions and Booleans help decide if something is `true` or `false`.
+- Use `if/else`, `switch`, or `ternarys` to decide what to do in a given situation.
+
+### Conditions and Comparisons
+- Conditions evaluate to `true` or `false`.
+- They do this with Comparison Operators and Logical Operators.
+  - Comparison Operators compare two values of the same type.
+  - Logical Operators compare the outcomes of two conditions.
+
+#### Comparison Operators:
+| Operator | Name             | Example        |
+|----------|------------------|----------------|
+| ===      | Equal To          | `'Hi' === 'Hi'`|
+| !==      | Not Equal To      | `6 !== 7`      |
+| >        | Greater Than      | `10 > 1`       |
+| <        | Less Than         | `3 < 5`        |
+| >=       | Greater or Equal  | `5 >= 5`       |
+| <=       | Less or Equal     | `2 <= 2`       |
+
+#### Logical Operators:
+| Operator | Name             | Example        |
+|----------|------------------|----------------|
+| &&       | AND               | `true && false`|
+| ||       | OR                | `true || false`|
+| !        | NOT               | `!true`        |
+
+### `if/else` statements
+- 3 parts: `if`, `else if`, `else`.
+- Starts checking at the top and stops checking when it hits a match.
+- Only 1 block is executed, so be specific first and get more general.
+
+#### Example:
+```javascript
+const happyBirthday = (age) => {
+  if (age === 25) {
+    return "Happy birthday, you can rent a car!";
+  } else if (age < 25) {
+    return "No rental cars on your birthday yet.";
+  } else {
+    return "Enjoy car rentals on your birthday!";
+  }
+}
+```
+
+## Simplify Your Logic
+### Don’t nest too much
+- Code should be readable.
+- Avoid deeply nested `if/else` statements.
+- Use guard clauses and ternarys.
```

Comment:
Define what a guard clause is.

"Use guard clauses and ternarys. A guard clause is an if statement that returns before subsequent has a chance to be executed."